client/tests/simulator/random.rs

```diff
@@ -0,0 +1,98 @@
+#![cfg(feature = "tpm-simulator-tests")]
+
+// This module is added to "mod.rs" to be included in the test. See "mod.rs" for details.
```

Comment:
Not sure this is a necessary comment; the comment about test structure and organization in the sibling `mod.rs` is probably sufficient IMHO.